Lesson_11_module/user_input.py

```diff
@@ -0,0 +1,12 @@
+def temperature_get() ->float:
+    temperature_str = input("Please enter degree of temperature to convert: ")
```

Comment:
The function shouldn't have input() inside. The function should have arguments that you should send when calling the function